ruby/vampires.rb

```diff
@@ -1,3 +1,4 @@
+
 puts "How many employees are you processing?"
   user_input = gets.to_i
 
```

Comment:
The allergies loop on line 35 should occur before making the full vampire evaluation.

You want to make sure that you have all of the information before deciding whether or not this employee is a vampire.